mindsdb/interfaces/skills/sql_agent.py

```diff
@@ -134,61 +132,24 @@ def get_table_info(self, table_names: Optional[List[str]] = None) -> str:
         If `sample_rows_in_table_info`, the specified number of sample rows will be
         appended to each table description. This can increase performance as demonstrated in the paper.
         """
-        try:
```

Comment:
@dusvyat, here I've switched using cache per table to:
- make it simplier
- get rid of using _database